os/board/common/Kconfig

```diff
@@ -31,20 +31,20 @@ config FLASH_MINOR
 
 config FLASH_PART_SIZE
 	string "Flash partition size list (in KBytes)"
-	default "16,48,192,32,512,2400,1536,1536,1400,8,512,"
+	default "16,48,192,32,512,1200,1200,1536,1536,1400,8,512,"
 	---help---
 		Comma separated list of partition sizes in KB.
 
 config FLASH_PART_TYPE
 	string "Flash partition type list"
-	default "none,none,none,none,none,none,none,none,smartfs,config,none,"
+	default "none,none,none,none,none,none,none,none,none,smartfs,config,none,"
 	---help---
 		Comma separated list of partition types that can be one of
 		followings: none, smartfs, config
 
 config FLASH_PART_NAME
 	string "Flash partition name list"
-	default "bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"
+	default "bl1,sssro,bl2,sssfw,wlanfw,os,apps,factory,ota,user,nvram,sssrw,"
```

Comment:
protected build is not a default.
Let's use two defaults with conditional.
```
default "xxxxxx" if !BUILD_PROTECTED
default "yyyyyy" if BUILD_PROTECTED
```

Reply:
ok. Done